xml-csharp/XMLDeserializationInCsharp/XMLDeserializationInCsharp/Program.cs

```diff
@@ -2,51 +2,82 @@
 
 namespace XMLDeserializationInCsharp
 {
-
     public class Program
     {
         static void Main(string[] args)
         {
+            #region Deserialization_Of_Class
             #region Simple Deserialization
-            var xmlData = "<Person>" +
-                        "<Name>John Doe</Name>" +
-                        "<Age>30</Age>" +
-                        "</Person>";
+            var xmlData = @"<Person>
```

Comment:
This works, but my suggestion was to make it a raw string literal, that way it won't have a ton of extra unnecessary whitespace in the string. Raw string literals start with """. [Here is a link](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-11.0/raw-string-literal) to the proposal documentation which explains it more. It is perfect for defining json and xml in code and preserving the formatting and spacing properly. Both VS and Rider handle it properly when formatting the code.

Reply:
Hi @jshergal Thank you for the detailed answer i have changed string to raw string literal now you can check